drivers/scheduler/openshift/openshift.go

```diff
@@ -332,7 +332,7 @@ func updateMaxUnavailableWorkerMCP() error {
 
 	maxUnavailable := os.Getenv("OCP_SURGE_UPGRADE_VALUE")
 	log.Info("Setting maxUnavailable value for worker MCP to [%s]", maxUnavailable)
-	patchData := fmt.Sprintf(`{"spec":{"maxUnavailable":"%s"}}`, maxUnavailable)
+	patchData := fmt.Sprintf(`{"spec":{"maxUnavailable":%s}}`, maxUnavailable)
```

Comment:
The patch command might fail if maxUnavailable is empty.

Reply:
Added return in case the value is empty.